### PR TITLE
Update Getting-Started-CUDA.md

### PR DESCRIPTION
**Pull request by dlwh**
Taking a guess at what needed to change. @Ivan-Zhou and @julian-q please lmk.

### CODE REVIEW

File: docs/Getting-Started-CUDA.md [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
 
 ## Running a job on Slurm
 
 ### Single Node
 
+Simple example, customize to your needs:
+
 ```bash
-srun --cpus-per-task=128 --gres=gpu:8 --job-name=levanter-multi-1 --mem=1000G --nodelist=sphinx[7-8] --open-mode=append --partition=sphinx --time=14-0 ~/src/levanter/scripts/run-slurm.sh python examples/gpt2_example.py --config_path config/gpt2_small.yaml
+srun --cpus-per-task=128 --gres=gpu:8 --job-name=levanter-multi-1 --mem=1000G --nodelist=sphinx7 --open-mode=append --partition=sphinx --time=14-0 ~/src/levanter/scripts/run-slurm.sh python examples/gpt2_example.py --config_path config/gpt2_small.yaml
```
Comment (Ivan-Zhou) [inline docs/Getting-Started-CUDA.md:41]: Note that I just tested in sphin6, which used CUDA 11.7. I installed jax 0.4.11. 

I had this issue when running Levanter. This is consistent with what I encountered on jagupard (P6000):

```
FAILED tests/test_backpack.py::test_backpack_predict - jaxlib.xla_extension.XlaRuntimeError: INTERNAL: nvlink exited with non-zero error code 256, output: nvlink fatal   : Input file '/tmp/ivanzhou/tempfile-sphinx6-efa86375-3475856-5fd80f1d5370f.cubin' newer than toolkit (118 vs 113)
```

**Review by Ivan-Zhou — APPROVED** [review]
The change to the doc LGTM. I just have a separate issue that is specific to the slurm environment.

File: docs/Getting-Started-CUDA.md (hunk 1 above)
Comment (Ivan-Zhou) [reply docs/Getting-Started-CUDA.md:41]: I don't think this is an issue of Levanter. This is more likely something between CUDA toolkit and Jax.

Comment (dlwh) [reply docs/Getting-Started-CUDA.md:41]: image

Yeah looks like we need to increase the support version range and/or require people to build form source on sphinx and/or get the sphinxes upgraded. We should probably do all three.

Comment (dlwh) [reply docs/Getting-Started-CUDA.md:41]: (that's a screenshot of the current jax readme)

**Pull request merged**